pkg/services/policer/policer.go

```diff
@@ -22,12 +23,41 @@ type nodeLoader interface {
 	ObjectServiceLoad() float64
 }
 
+type objectsInWork struct {
+	m    *sync.RWMutex
```

Comment:
Why did u decide to hold pointer instead of mutex itself? Is there any case when we set pointer to already allocated mutex somewhere else?

Reply:
> Why did u decide to hold pointer instead of mutex itself?

cause i do not use a pointer to `objectsInWork` in the policer

> Is there any case when we set pointer to already allocated mutex somewhere else?

do you mean in that PR? or somewhere else in the repo?

Reply:
Then why did u decide to avoid using `objectsInWork` by a pointer?

Reply:
no real reason. maybe to store struct with pointers and not to have a pointer to pointer (map). changed

do you have any pros to say that one approach is better than another? thanks!

Reply:
In terms of source code: we don't need to remember to init pointer.
In terms of performance: I've benchmarked similar case once, it showed very close performance.

I adhere to the approach that if type's method changes value's internal state by design, then this method is called by a pointer.